zingolib/src/wallet/confirmation_status.rs

```diff
@@ -16,9 +22,9 @@ fn u32_height_or_placeholder(option_blockheight: Option<BlockHeight>) -> u32 {
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub enum ConfirmationStatus {
     Local,
```

Comment:
I tried to think of other idents than "Local"..   but I think in the end I decided I liked "Local" more than the alternatives I came up with.